libraries/entities/src/DiffTraversal.cpp

```diff
@@ -187,27 +159,28 @@ DiffTraversal::Type DiffTraversal::prepareNewTraversal(const ViewFrustum& viewFr
     // external code should update the _scanElementCallback after calling prepareNewTraversal
     //
     _currentView.usesViewFrustum = usesViewFrustum;
+    float lodScaleFactor = powf(2.0f, lodLevelOffset);
```

Comment:
The `lodScaleFactor` is just a power of two.  As the `lodLevelOffset` increments `lodScaleFactor` doubles.  This is effectively identical behavior to the old `renderAccuracy` and LOD cuttoff distance stuff but with less obfuscation.